axon/networkbase.go

```diff
@@ -419,8 +419,8 @@ func (nt *NetworkBase) Build() error {
 			nt.LayClassMap[cl] = ll
 		}
 	}
-	nt.Neurons = make([]Neuron, totNeurons, totNeurons) // never grows
-	nt.Prjns = make([]AxonPrjn, totPrjns, totPrjns)
+	nt.Neurons = make([]Neuron, totNeurons) // never grows
```

Comment:
If you explicitly specify the capacity as the 3rd arg, then it allocates exactly that size array -- otherwise it allocates extra to allow for growth.  so for non-dynamic arrays it should be better to specify this 3rd arg.

Reply:
Docs are saying:
>  The size specifies the length. The capacity of the slice is equal to its length. A second integer argument may be provided to specify a different capacity
So it sounds like there's no overallocation on initialization, which also how Python handles it's lists (most lists never change their length).

IntelliSense highlighted it as being superflous, that's why I changed it, both was are fine by me :)

Reply:
ah ok, I had assumed it might do proactive pre-allocation of larger capacity but apparently it only does that when existing capacity is exceeded.